### PR TITLE
update circleci to push to staging

### PR DESCRIPTION
**Pull request by ryanwoldatwork**

### CODE REVIEW

File: bin/deploy-production-ci.sh [hunk 1]
```diff
@@ -2,15 +2,13 @@
...
 set -e
 
-# Install CF
-curl https://s3-us-west-1.amazonaws.com/cf-cli-releases/releases/v6.46.1/cf-cli_6.46.1_linux_x86-64.tgz | tar xzvf -
-sudo mv cf /usr/local/bin/cf
-
 # Log into cloud.gov
 cf api api.fr.cloud.gov
 cf login -u $CF_USERNAME -p $CF_PASSWORD -o gsa-opp-analytics -s analytics-dev
+# cf login -u $CF_PRODUCTION_SPACE_DEPLOYER_USERNAME -p $CF_PRODUCTION_SPACE_DEPLOYER_PASSWORD -o gsa-opp-analytics -s analytics-dev
```
Comment (btylerburton) [inline bin/deploy-production-ci.sh:8]: did you want to include this?

Comment (ryanwoldatwork) [reply bin/deploy-production-ci.sh:8]: thanks for flagging.
yes. i plan to update this to move to a Service Account.

**Review by btylerburton — APPROVED** [review]
👍🏼

File: bin/deploy-production-ci.sh (hunk 1 above)
Comment (ryanwoldatwork) [reply bin/deploy-production-ci.sh]: updated

File: bin/deploy-production-ci.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# Log into cloud.gov
+cf api api.fr.cloud.gov
+cf login -u $CF_PRODUCTION_SPACE_DEPLOYER_USERNAME -p $CF_PRODUCTION_SPACE_DEPLOYER_PASSWORD -o gsa-opp-analytics -s analytics-dev
```
Comment (btylerburton) [inline bin/deploy-production-ci.sh:7]: you probably don't want `analytics-dev` as the space

Comment (ryanwoldatwork) [reply bin/deploy-production-ci.sh:7]: agreed. but to address later, as its a larger infrastructure change

https://github.com/18F/analytics-reporter/issues/675

**Review by btylerburton — APPROVED** [review]

**Pull request merged**